src/assets/icons/core-symbol.svg

```diff
@@ -1 +1 @@
.8.5-.1.9-.1 1.3-.1 2.3 0 3.7.9 5.5 2.1l.9.6c.7.5 1.4.7 1.9.7.9 0 1.4-.6 2.1-1.7.8-1.1 1.8-2.4 3.5-2.6 1.8-.2 3.3.7 4.6 1.5 1 .6 1.9 1.2 2.8 1.2.7 0 1.5-.4 2.3-1.2 1-1 1.9-1.7 2.9-1.7.7 0 1.4.4 2.3 1.2l.2.2c1.8 1.7 2.3 2.2 4.5 1 1.7-.9 3.9-1.5 5.7-1.5 3.1 0 5 1.6 5.1 4.3l-.3.1-.5.2c-.2.1-.3.1-.5.2-2.5.8-9.7 4.8-10.4 5.4-2.4 1.8-4.7 4.2-6.4 6.6-6.6 9.4-6.5 22.8-4.3 31.5 1.1 4.5 2.6 9.8 5.3 14.8 1.2 2.2 2.6 4.2 4.1 6.1l.2.2c3.7 4.4 4.5 4.6 4.7 4.8.1.1.2.2.3.4 0 .3-.3.5-.5.6-.3.2-.7.3-1 .4s-.6.1-1 .2-.8.1-1.1.1c-1.7.1-3.3.2-5 0-1-.1-2.1-.2-3.1-.4-1.2-.2-3.8-.8-5.3-1.6l-1 .5c-1.5 1-4.2 2.2-4.6 2.3-1.6.3-3.1.6-4.6.6zm-11.5-6.5c.9.8 3 2.5 3.2 2.7 1.8 1.1 3.4 1.9 5 2.3 2.4.6 5 .5 7.4-.3.3-.1 3.2-1.3 4.5-2.2l-.3-.6c-1.1-2.2-2.6-5.1-3.3-7.3-.2-.5-.3-.9-.5-1.3-.8-2.3-1.5-4.2-1.5-7.4v-2.8l6.5 17.4c.2.3.3.7.4 1 .5.7 3.2 1.5 5.5 1.9 3.8.7 7.3.7 10 0-.8-.7-2.1-2-4.2-4.5l-.2-.2c-1.6-1.9-3-4-4.2-6.2-2.7-5-4.2-10.4-5.4-15-2.2-8.8-2.3-22.6 4.5-32.3 1.8-2.5 4.1-5 6.6-6.8.6-.5 8.7-4.9 11.1-5.5-.4-2.3-2.3-2.8-3.9-2.8s-3.6.5-5.2 1.3c-2.8 1.5-3.8.6-5.7-1.1l-.2-.2c-.6-.6-1.1-.9-1.6-.9-.6 0-1.3.6-2.2 1.4-1 1-2 1.4-3 1.4-1.2 0-2.3-.7-3.4-1.4-1.2-.8-2.4-1.6-3.9-1.4-1.4.2-2.1 1.1-2.9 2.2-.8 1-1.6 2.1-2.9 2.1-.7 0-1.5-.3-2.4-.9l-.9-.6c-1.8-1.2-3-2-4.9-2-.4 0-.7 0-1.1.1-1.6.2-2.7.7-3.5 1.5-.4.4-.7 1-1 1.5-.2.4-.4.8-.6 1.1 2.2 3.6 3.5 7.8 3.9 10.4.7 4.5-.4 7-1.5 9.4l-.8 1.8L29 52.7c-1.1-.6-2.1-1.3-3-2-2.2-1.6-4.3-3-7.4-3.3-1.4-.1-2.8-.2-4.3-.2-1.3 0-2.4.1-3.4.2-.6.1-4.7 1.7-5.5 2.5 2 1.6 4.3 3 6.5 4.3l1 .6c2.3 1.4 4.7 2.9 6.8 4.8 4.7 4.1 7.4 10.2 8.2 18.5.4 4.4.8 9.5.1 14.5-.7 4.6-1.7 9.9-3.7 14.1l5.1-4.6c.5-.5 1.2-1 1.7-1.4l2.7-2.2-3.6 5.4c-.8.7-1.4 1.3-2.1 2.3s-1.7 1.8-2.5 2.4-1.8 1-3.1 1.1c-.6.8-1.2 1.5-1.9 2.1-1.2 1-2.3 2.1-3.5 3.2-1.5 1.4-3 2.9-4.6 4.2.5.2 1.2.5 2.3.9.7.3 1.4.6 1.6.6 2.7.8 5 1.2 7.2 1.1 2.4-.1 4.3-1.4 6.2-2.7 1.1-.8 2.2-1.5 3.5-2.1l5.6-21.2v3.9c0 3.8-.3 5.5-.6 7.4-.2 1.2-.8 2.7-1.3 4.1-.3.7-.6 1.4-.7 2-.3 1-.6 1.7-1.1 2.5-.3.3-.6.8-.9 1.5zm16-10.1c.3 1.1.6 2 1 3.2.2.4.3.9.5 1.3.4 1.2 1 2.6 1.7 4zm-12.6-2.9l-1.7 6.5v-.1c.5-1.3 1.1-2.8 1.3-3.9.2-.8.3-1.6.4-2.5zm-9.5.3l-4.1 3.7c.4-.1.7-.3 1-.5.7-.6 1.7-1.3 2.3-2.1.2-.4.5-.7.8-1.1zm2-50.7L32.3 70c.8-1.9 1.3-4.1.8-7.5-.3-2.2-1.2-5.6-2.9-8.7zm26.7 58.8c-.4-.9-.6-1.8-.8-2.7l-.2-.7-.3-1.2c0-.1-.1-.2-.1-.3-.3-.8-.7-1.5-1-2.1-.4-.8-.9-1.6-1.2-2.5-1.2-3.7-2-6.8-2-10.4v-3.6l5.3 18.8c.1.3.2.7.3 1l.9 3.3zM45 79.9l2.8-6.2c.3-1 .5-2.3.7-3.6.2-1.4.4-2.7.7-3.5.9-2.4 3-3.9 4.9-5.3 1-.7 1.9-1.4 2.7-2.2l2.6-2.5L49.5 74c-.4 1.3-2.2 4.8-3.3 5.3zm10.4-18.2c-.2.2-.5.3-.7.5-1.9 1.4-3.7 2.7-4.6 4.9-.3.7-.5 2-.7 3.3-.1.9-.3 1.8-.5 2.6zm3.2-2.6l.7-2.3c.6-1.8 2.3-6.1 5.5-6.3l1.2-.1zm4.7-6.9c-.6.4-1.2 1.1-1.7 1.9zm15.2-37.9c-5.5.1-11.5.4-16.6 3.2-2.4 1.3-4.1 3-6.3 4.6-1.1.8-2.3 1.6-3.6 2.2-1.3.7-2.9.8-4.1 1.5l.6 1.5c1.7-.5 2-.7 3.7-.8s3.3-.4 5.1-.2c1.7.2 3.7.1 5.4.6 1.9.5 4.4.5 6.3.7 3.9.3 8.7-.6 12.6-1.4 2.8-.6 6.6-1.8 8.7-3.9.9-.9 1.4-1.8 2.5-2.8 1-1 2.6-2.9 3.4-4s1.4-1.8 2.2-3c.6-1 1.8-3.2 1.8-4.5-5.3 5.9-14.5 6.2-21.7 6.3z"/></svg>
```

Comment:
Please, do not optimise icons. It's better to have sources because sometimes it's not possible to edit icon anymore if it's optimised. CORE logo is such example.

[Check it here](https://github.com/oacore/design/blob/master/package.json#L87)